VisualizationBase/src/utils/InfoJavascriptFunctions.cpp

```diff
@@ -60,4 +61,24 @@ void InfoJavascriptFunctions::jumpToObject(QString fullName)
 		}
 }
 
+void InfoJavascriptFunctions::hideLayer(QString infoNodeKey, QString layerName)
+{
+	auto node = InfoNode::infoNode(infoNodeKey);
+	if (!node) return;
+	node->setEnabled(layerName, false);
```

Comment:
This is the only statement that is different between this method and the one below `moveLayer`. Please combine these two methods into one `updateLayer`. That method should take the `infoNodekey` the `layerName` and a new enum type `updateType` that has the values 'moveUp, moveDown, hide, show`